src/style.css

```
@@ -0,0 +1,156 @@
+/* stylelint-disable-next-line no-empty-source */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  --main-text-color: #060b35;
+}
+
+.product {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  flex-direction: column;
+  border: rgb(245, 245, 245) solid 0.5px;
+  border-radius: 5px;
+  max-width: 199px;
+  max-height: 407px;
+}
+
+.title {
+  display: flex;
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-bottom: 4px;
```

```suggestion
  margin: 40px 16px 4px;
```

replace with margin shorthand